styles2.css

```diff
@@ -657,3 +542,267 @@ footer > div.contentline {
   margin: 21px auto 8px auto;
   padding: 0;
 }
+
+.desktop {
+  display: none;
+}
+
+/* Desktop version */
+
+@media only screen and (min-width: 768px) {
+  main {
+    width: 100%;
+    background-image: url("leftcf.svg"), url("middlecf.svg"), url("rightcf.svg");
+    background-repeat: no-repeat;
+    background-position: bottom left, bottom 84.15px left 274px, bottom right;
+    background-size: 236px 261px, 130px 105px, 316px 450px;
+  }
+
+  section#main {
+    background-size: cover;
+    background-position: top;
+    width: 100%;
+    height: auto;
+  }
+
+  header#smallScreen {
+    display: none;
+  }
+
+  nav.navigation {
+    display: block;
+    width: 100%;
+    height: 72px;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  nav.navigation > ul {
+    width: 255px;
+    height: 40px;
+    margin: auto;
+    display: flex;
+    flex-direction: row;
+    align-items: space-between;
+    gap: 8px;
+    padding: 0;
+  }
+
+  nav.navigation > ul > li {
+    padding: 10px 12px;
+  }
+
+  section.headline {
+    height: auto;
+    width: 64%;
+    max-width: 920px;
+    margin: 0 auto auto auto;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+  }
+
+  h1 {
+    margin: 0 auto;
+  }
+
+  section.headline > header {
+    margin-bottom: 24px;
+  }
+
+  .contact-form > p {
+    font-size: 30px;
+    line-height: 52px;
+    height: 256px;
+    margin: 0;
+    width: 436px;
+    padding-top: 20px;
+  }
+
+  .content > p,
+  article.text > p {
+    height: auto;
+  }
+
+  #main section.icon {
+    margin: 24px auto 74px auto;
+  }
+
+  .contact-form button {
+    margin-left: 0;
+  }
+
+  section.headline > button {
+    margin-bottom: 184px;
+    margin-top: 0;
+  }
+
+  /* Recent work section */
+
+  section.recentWork {
+    width: auto;
+    height: auto;
+    padding: 84px 8px;
+    display: grid;
+    grid-template-columns: 369px 369px 369px;
+    align-items: center;
+    margin-left: auto;
+    margin-right: auto;
+    justify-content: space-between;
+    row-gap: 24px;
+    column-gap: 32px;
+    margin-bottom: 28px;
+  }
```

Comment:
- [ ] You have done great implementing the cards using `grid` :+1: . However, the recent works section is not responsive for widths below `1244px`. This leads to a `horizontal scroll` Kindly ensure that the width reduces as the responsiveness width reduces. You may consider the use of `max-width` or `percentage width`.
-Your design :sunglasses: 
![Screenshot from 2023-01-20 06-14-49](https://user-images.githubusercontent.com/99394374/213611251-095c0833-fbbb-48a7-9e7b-f580ef277623.jpg)

- Figma design :sunglasses: 
![Screenshot from 2023-01-20 06-12-41](https://user-images.githubusercontent.com/99394374/213611371-2817a454-0966-4378-80a5-117e38a61461.png)